tests/generation/FileGlobs.py

```diff
@@ -96,3 +96,27 @@ def test_get_project_files_gitignore(self):
 
         os.remove(".gitignore")
         os.chdir(orig_cwd)
+
+
+    def test_get_project_files_ci_mode(self):
+        orig_cwd = os.getcwd()
+        os.chdir(os.path.dirname(os.path.realpath(__file__)))
+        os.makedirs("file_globs_ci_testfiles", exist_ok=True)
+        os.chdir("file_globs_ci_testfiles")
+
+        os.makedirs("src", exist_ok=True)
+        os.makedirs("ignore_dir", exist_ok=True)
+        open(os.path.join("src", "file.c"), "w").close()
+        open("root.c", "w").close()
+        open(os.path.join("ignore_dir", "src.c"), "w").close()
+        open(os.path.join("ignore_dir", "src.js"), "w").close()
+
+        with suppress_stdout():
+            res, _ = get_project_files(self.log_printer, self.printer, os.getcwd()
+                , True)
+            self.assertIn(os.path.join(os.getcwd(), "src", "file.c"), res)
+            self.assertIn(os.path.join(os.getcwd(), "root.c"), res)
+            self.assertIn(os.path.join(os.getcwd(), "ignore_dir/src.c"), res)
```

Comment:
'/' should be `os.sep`
Doing file tests without AppVeyor (https://github.com/coala/coala-quickstart/issues/22) is risky.